.github/workflows/ci-riscv.yml

```diff
@@ -154,7 +154,7 @@ jobs:
 
       # Setup QEMU for RISC-V platform support
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@53851d14592bedcffcf25ea515637cff71ef929a # v3.0.0
+        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.0.0
```

Comment:
Actually version v3.6.0: https://github.com/docker/setup-qemu-action/releases/tag/v3.6.0

Reply:
Looks like the bot ignores the comment if it does not match version represented by githash.